### ISSUE TITLE
Gens preprocessing

### ISSUE THREAD

**Issue by raysloks** [Clinical-Genomics/MIP#1970]
Hello, I'm adding [Gens](https://github.com/Clinical-Genomics-Lund/gens)'s preprocessing to MIP.

I'm currently using a container with htslib and Gens's perl-scripts.
Should I add this container to the install-process, and if so, how should I go about it?

**Comment by jemten** [comment]
Hello there, 
I think we should try to add it to [nf-core raredisease](https://github.com/nf-core/raredisease) instead. After next release MIP will mainly receive patches.
/Anders

**Comment by raysloks** [comment]
I agree, that'd be the smoothest approach.
However, the instructions I received were to implement it into MIP, as the nf-core pipeline was considered a bit too far from being used in production.
@vwirta is this still the case? Or should we reconsider?

[Project planning](https://github.com/Clinical-Genomics/project-planning/issues/334) for convenience.

**Comment by jemten** [comment]
Yeah, the nf-core pipeline is still far away from being ready and implemented in production. All depends on how soon we want this running in production. My gut feeling is that if this is to be used in prod in Q1 or Q2 2022 it needs to go into MIP otherwise it should go into the nf-core pipeline.

**Comment by dnil** [comment]
It is essentially working components, in prod in Lund all the way to Scout, though they require a little tweaking to fit here. Not super urgent, but convenient for the cyto-genomics users. Since @raysloks is already well underway, wouldn't it make sense to just add it to MIP anyway? But agreed it is not necessarily a patch level addition. 😅

**Comment by jemten** [comment]
done
